solidity/migrations/2_deploy_contracts.js

```diff
@@ -57,6 +58,11 @@ const all = [
 
 module.exports = (deployer, network, accounts) => {
   deployer.then(async () => {
+    if (network == "keep_dev" || network == "local") {
```

Comment:
Are you missing the "development" network? We've got it [here](https://github.com/keep-network/tbtc/blob/master/solidity/truffle-config.js#L52) in truffle-config.